--- a/techpack/display/msm/Makefile
+++ b/techpack/display/msm/Makefile
@@ -132,14 +132,9 @@ msm_drm-$(CONFIG_DSI_PARSER) += dsi/dsi_parser.o \
 #endif
 
 msm_drm-$(CONFIG_DRM_MSM) += \
-	msm_atomic.o \
-	msm_fb.o \
-	msm_iommu.o \
-	msm_drv.o \
-	msm_gem.o \
-	msm_gem_prime.o \
-	msm_gem_vma.o \
-	msm_smmu.o \
+	msm_smmu.o \
 	msm_prop.o \
 	msm_notifier.o\
 
@@ -147,7 +142,7 @@ obj-$(CONFIG_DRM_MSM)	+= msm_drm.o
 
 #ifdef OPLUS_BUG_STABILITY
 ###########################################################################################
-#build oplus so
+#build oplus helpers
 msm_drm-$(CONFIG_DRM_OPLUS) += ../oplus/oplus_dsi_support.o \
 	../oplus/oplus_display_private_api.o \
 	../oplus/oplus_dc_diming.o \
